test/test_fapi.py

```diff
@@ -530,11 +526,21 @@ def test_nv_write_read(self, nv_ordinary):
         assert log == ""
 
     def test_nv_increment(self, nv_increment):
+        # TODO initial increment should not be necessary, check in with upstream
```

Comment:
Did you file a bug?

Reply:
No, I think this is not a bug, but another TPM spec quirk. [III. Command Spec](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part3_Commands_pub.pdf), 31.13 TPM2_NV_Read:

> If the NV Index has been  defined but the `TPMA_NV_WRITTEN` attribute is `CLEAR`, then this command 
shall return `TPM_RC_NV_UNINITIALIZED` [...].

That is, if the index was defined but never written to (here incremented), reading fails. That's counter-intuitive, so we might want to add a more verbose warning.